pkg/static/login.js

```diff
@@ -205,13 +230,13 @@
             const args = [].join.call(arguments, ": ");
 
             if (!window.CSS || !window.CSS.supports.apply(this, arguments)) {
-                disableLogin(args);
+                disableLogin(args, "bypass");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17774-20221017-161558-ae4aa609-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)